models/pull.go

```diff
@@ -349,7 +350,7 @@ func (pr *PullRequest) GetDefaultSquashMessage() string {
 
 // GetGitRefName returns git ref for hidden pull request branch
 func (pr *PullRequest) GetGitRefName() string {
-	return fmt.Sprintf("refs/pull/%d/head", pr.Index)
+	return fmt.Sprintf(git.PullPrefix+"%d/head", pr.Index)
```

Comment:
Is there a reason why this isn't
```suggestion
	return fmt.Sprintf("%s%d/head", git.PullPrefix, pr.Index)
```